crates/compiler/can/src/expr.rs

```diff
@@ -197,6 +197,9 @@ pub enum Expr {
     /// tuple or field accessor as a function, e.g. (.foo) expr or (.1) expr
     RecordAccessor(StructAccessorData),
 
+    /// tuple or field updater as a function, e.g. (&foo) record value
+    RecordUpdater(StructUpdaterData),
```

Comment:
```suggestion
    /// Field updater as a function, e.g. (&foo) record value
    RecordUpdater(StructUpdaterData),
```

This only applies to records, right?